app/views/api/v1/events_calendar/show.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+json.events do
+  json.array!(@events) do |event|
+    invitation = event.invitations.find_by(user_id: current_api_v1_user.id)
```

Comment:
This should not be done here! Variables like this should be set in the controller